src/js/pages/chapters/ChaptersPage.js

```diff
@@ -49,11 +74,16 @@ export class ChapterPage extends Component {
 
           <CardsContainer>
             {chapters.map((chp, index) =>
-              <ChapterCard key={index} {...chp} {...this.props} />)}
+              <ChapterCard key={index} {...chp}
+                {...this.props} viewingComments={this.state.viewingComments}
+                chapterComments= {this.props.chapterComments}
```

Comment:
add chapterComments, on line 59 and you can reduce the length in line 79  so the code will look cleaner and we avoid repetition.

Reply:
good catch